tests/python/open_data/gbm/test_xgb_sklearn_wrapper.py

```diff
@@ -2,12 +2,17 @@
 import numpy as np
 import logging
 import pandas as pd
+import platform
 
 print(sys.path)
 
 
 logging.basicConfig(level=logging.DEBUG)
 
+# TODO: remove when support NCCL on ppc64le platform
```

Comment:
nccl supported on cuda9.2 or cuda10 for ppc I think already.  Dealt with that before, we can chat if not clear.